recipe-app/View/RecipeListView.swift

```diff
@@ -8,6 +8,7 @@
 import UIKit
 
 class ViewController: UIViewController {
```

Comment:
Can be Final and let's keep the file names aligned with the class names. `RecipeListView` and `ViewController` doesn't seem to be matching.